docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -1111,28 +1116,26 @@ Selectors you write can return _any_ value. You can use it to “just” select
 nodes from the root tree (first two examples above), but you can also return
 computed values, like in the last two examples.
 
-💡 **Important**
-One important rule is that selector functions **must return a stable result** to
-be efficient. This means calling the same selector twice with the same argument
-should return two results that are _referentially equal_. Special care needs to
-be taken when filtering or mapping over arrays, or when returning object
-literals, because those operations create new array or object instances on every
-call. (Why this is is detailed
+💡 **Important** One important rule is that selector functions **must return a
+stable result** to be efficient. This means calling the same selector twice with
+the same argument should return two results that are _referentially equal_.
+Special care needs to be taken when filtering or mapping over arrays, or when
+returning object literals, because those operations create new array or object
+instances on every call. (Why this is is detailed
 [in the next section](#selectors-subscribe-to-updates).)
 
 Some examples of what "a stable result" means:
 
-- ✅ `(root) => root.animals` is stable 👍
-  Liveblocks guarantees this. All nodes in the Storage tree are stable
-  references as long as their contents don’t change.
+- ✅ `(root) => root.animals` is stable 👍 Liveblocks guarantees this. All nodes
+  in the Storage tree are stable references as long as their contents don’t
+  change.
```

Comment:
I think these have been lost in translation here somewhere along the lines. I once deliberately put a newline after the "👍" using the Markdown syntax by ending that line with `  ` (two spaces) before the newline. Prettier _should_ respect that and keep the hard line break in there. Not sure if something changes in Prettier, or the spaces got removed at one point, or our MDX plugin doesn't respect those.

Reply:
Gotcha! It's still possible to do this with `<br />` but I've used some other elements for a better style.